<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fx3deviceconfig</class>
 <widget class="QDialog" name="fx3deviceconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>381</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>fx3deviceconfig</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Choose Platform</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Platform:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="platformChoosecomboBox">
        <item>
         <property name="text">
          <string>Grabber</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">UVC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">RTSPM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">HDMI</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="choosePlatformpushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Grabber Serial Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="serialNumberlabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Connect Grabber:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="connectgrabbersncomboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Choose Grabber to Open:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="anyradioButton">
          <property name="text">
           <string>ANY</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="pos1radioButton">
          <property name="text">
           <string>Grabber 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="pos2radioButton">
          <property name="text">
           <string>Grabber 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="pos3radioButton">
          <property name="text">
           <string>Grabber 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="pos4radioButton">
          <property name="text">
           <string>Grabber 4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="savepushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>BIND</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Grabber Parameter Setting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Upload Delay(clock):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fpgaDelayspinBox">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="value">
           <number>110</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Packet Size(64KB):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="packetSizespinBox">
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>64</number>
          </property>
          <property name="value">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="setGrabberParameterpushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
